external-tests/clone_repo.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+set -x
+
+if [ "$TESTS_REPO" == "" ]; then
+	echo "[ERROR]: No TESTS_REPO provided"
+	exit 1
+fi
+
+if [ "$TESTS_LOCATION" == "" ]; then
+	echo "[ERROR]: No TESTS_LOCATION provided"
+	exit 1
+fi
+
+# detect latest master hash of ocs-ci if we're not pinned to a specific commit hash
+if [ -z "$TESTS_TARGET_HASH" ]; then
```

Comment:
@davidvossel this ^^^^ is the pinning logic (mutuated by the script you showed me)

Reply:
yeah, but if you have multiple test suites... you can't pin each one individually. it would only work for one.

Reply:
I was thinking about having it in the feature's test.sh (which eventually calls this script)